src/index.ts

```diff
@@ -63,7 +63,7 @@ const argv = minimist(process.argv, {
 
 const CODE_LOCATION = resolve(
   process.cwd(),
```

Comment:
Isn't process.cwd redundant here? According to the docs [1] the cwd is prepended to any relative path.

https://nodejs.org/docs/latest/api/path.html#path_path_resolve_paths

Reply:
It is redundant. I thought the docs meant `cwd` would be used as a path instead of `cwd` being prepended.